### PR TITLE
#182234874 (Feature): Implement landing page

### PR DESCRIPTION
**Pull request by itisWasp**
### What does this PR do?

Implement landing page

### Description of Task to be completed?
- Use material UI for components design
- Create reusable layouts like footer and navbar as they will be used in a a lot of pages
- Make sure you separate components and make them as reusable as possible

### How should this be manually tested?

To test the changes manually:

* clone repo on your local machine
* switch to ``ft-landing-page-182234874`` branch
* start the application with `yarn dev` command
* on the opened page, scroll to view the page

### What to expect after running the commands:

On the opened page, scroll to browse through the components of the landing Page

### Any background context you want to provide?

- N/A

### Useful links

N/A

### What are the relevant pivotal tracker stories?

* [182234874](https://www.pivotaltracker.com/story/show/182234874)

### Screenshots (if appropriate)

- N/A

### Questions:

- N/A

### CODE REVIEW

File: src/components/home/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (ericus123) [inline src/components/home/Button.jsx]: why is this rule disabled everywhere ?

File: src/components/home/Home.jsx [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
   return (
-    <div className="welcome">
-      <b>
-        Welcome to <i>BAREFOOT NOMARD.</i>
-      </b>
-    </div>
+    <ProductLayout
+      sxBackground={{
+        backgroundImage: `url(${backgroundImage})`,
+        backgroundColor: '#7fc7d9', // Average color of the background image.
```
Comment (ericus123) [inline src/components/home/Home.jsx]: don't hardcode colors.

File: src/components/home/Typography.jsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/require-default-props */
```
Comment (ericus123) [inline src/components/home/Typography.jsx]: why disable these rules ???

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: src/components/home/AppBar.jsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+const rightLink = {
+  fontSize: 16,
+  color: 'common.white',
+  ml: 3,
+};
+
+function AppAppBar() {
+  return (
+    <div>
+      <AppBar position="fixed" sx={{ bgcolor: '#1D97A7' }}>
```
Comment (ericus123) [inline src/components/home/AppBar.jsx]: this bgcolor should be a variable, not hardcoded like this.
do the same where you have hardcoded colors.
make sure they are predefined

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: src/components/home/Button.jsx (hunk 1 above)
Comment (itisWasp) [reply src/components/home/Button.jsx]: @ericus123 The way I was destructuring props works but Eslint was giving errors.

File: src/components/home/Typography.jsx (hunk 3 above)
Comment (itisWasp) [reply src/components/home/Typography.jsx]: Same Issue here.

**Review by Procech20 — DISMISSED** [review]
LGTM

**Review by Robertishimwe — DISMISSED** [review]

**Review by niyojuste — APPROVED** [review]

**Review by Robertishimwe — APPROVED** [review]

**Review by ericus123 — APPROVED** [review]

**Pull request merged**